website/content/en/docs/getting-started/_index.md

```diff
@@ -156,10 +156,11 @@ eksctl. Thus, we don't need the helm chart to do that.
 ```bash
 helm repo add karpenter https://charts.karpenter.sh
 helm repo update
-helm upgrade --install --skip-crds karpenter karpenter/karpenter --namespace karpenter \
-  --create-namespace --set serviceAccount.create=false --version 0.4.1 \
+helm upgrade --install karpenter karpenter/karpenter --namespace karpenter \
+  --create-namespace --set serviceAccount.create=false --version 0.4.2 \
```

Comment:
Can we do 0.4.3 here since that's the latest release?

Reply:
oh man, where have I been. Yep